excise_management/data/ir_model_fields.xml

```diff
@@ -257,4 +257,80 @@ for template in self:
         <field name="relation">uom.uom</field>
         <field name="readonly" eval="True"/>
     </record>
+    <record id="account_tax_deposit_field" model="ir.model.fields">
+        <field name="name">x_is_deposit</field>
+        <field name="field_description">Is Deposit</field>
+        <field name="model_id" ref="account.model_account_tax"/>
+        <field name="ttype">boolean</field>
+    </record>
+    <record id="sale_order_line_excise_free_unit_price_field" model="ir.model.fields">
+        <field name="name">x_excise_free_unit_price</field>
+        <field name="field_description">Excise Free Unit Price</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="ttype">monetary</field>
+        <field name="depends">price_unit, tax_ids</field>
+        <field name="compute"><![CDATA[for record in self:
+        record['x_excise_free_unit_price'] = record.price_unit - record.product_id.x_excise_amount]]></field>
+        <field name="readonly" eval="True"/>
+        <field name="store" eval="False"/>
+        <field name="currency_field">currency_id</field>
+    </record>
+    <record id="sale_order_line_total_excises_field" model="ir.model.fields">
+        <field name="name">x_total_excises</field>
+        <field name="field_description">Excises</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="ttype">monetary</field>
+        <field name="depends">product_id, product_uom_qty</field>
+        <field name="compute"><![CDATA[for record in self:
+        record['x_total_excises'] = record.product_id.x_excise_amount * record.product_uom_qty if not record.order_id.fiscal_position_id.x_is_fiscal_deposit else False]]></field>
+        <field name="readonly" eval="True"/>
+        <field name="currency_field">currency_id</field>
+    </record>
+    <record id="sale_order_line_total_deposits_field" model="ir.model.fields">
+        <field name="name">x_total_deposits</field>
+        <field name="field_description">Deposits</field>
+        <field name="ttype">monetary</field>
+        <field name="model_id" ref="sale.model_sale_order_line"/>
+        <field name="depends">price_unit, product_uom_qty, tax_ids.x_is_deposit</field>
```

Comment:
we won't update on SOL if a tax is modified, only if the tax is (un)assign to the SOL
```suggestion
        <field name="depends">product_uom_qty, tax_ids</field>
```